exonum/src/blockchain/service.rs

```diff
@@ -172,6 +172,12 @@ pub trait Service: Send + Sync + 'static {
         Value::Null
     }
 
+    /// A service execution. This method is invoked for each service after execution
+    /// of all transactions in the block but before `handle_commit` handler. The order of
```

Comment:
Nitpick: I'd add a paragraph break after "This method is invoked..." to keep the first paragraph (which is used as summary in `Service` implementations) reasonably short.